[BUG] cannot update-crd

**What is your environment(Kubernetes version, Fluid version, etc.)**
fluid-master 971309e89

**Describe the bug**
when update-crd:
```bash
make update-crd
GO111MODULE=off /Users/qiulingwei/Projects/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: unable to parse string: invalid syntax (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: unable to parse string: invalid syntax (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: unable to parse string: invalid syntax (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: invalid char escape (at <input>:1:1)
/Users/qiulingwei/Projects/go/src/github.com/fluid-cloudnative/fluid/api/v1alpha1/alluxioruntime_types.go:144:2: unable to parse string: invalid syntax (at <input>:1:1)
Error: not all generators ran successfully
```
the mistake happened at:
```go
	// Quotas in this list will be set to paths with the same order in Path.
	// For example, with Path defined with "/mnt/cache1,/mnt/cache2" and QuotaList set to "100Gi, 50Gi",
	// then we get 100GiB cache storage under "/mnt/cache1" and 50GiB under "/mnt/cache2".
	// Also note that num of quotas must be consistent with the num of paths defined in Path.
	// +optional
	// +kubebuilder:validation:Pattern:="^((\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+)))),)+((\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?)$"
	QuotaList string `json:"quotaList,omitempty"`
```
**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/charts/fluid/fluid/Chart.yaml
+++ b/charts/fluid/fluid/Chart.yaml
@@ -18,7 +18,7 @@ version: 0.6.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 0.6.0-6d1715f
+appVersion: 0.6.0-d04cd4d
 home: https://github.com/fluid-cloudnative/fluid
 keywords:
   - category:data

--- a/charts/fluid/fluid/crds/data.fluid.io_alluxioruntimes.yaml
+++ b/charts/fluid/fluid/crds/data.fluid.io_alluxioruntimes.yaml
@@ -522,7 +522,7 @@ spec:
                           storage under "/mnt/cache1" and 50GiB under "/mnt/cache2".
                           Also note that num of quotas must be consistent with the
                           num of paths defined in Path.
-                        pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?,((\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?)+$
+                        pattern: ^((\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+)))),)+((\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?)$
                         type: string
                     required:
                     - mediumtype

--- a/charts/fluid/fluid/crds/data.fluid.io_databackups.yaml
+++ b/charts/fluid/fluid/crds/data.fluid.io_databackups.yaml
@@ -106,8 +106,7 @@ spec:
               description: Conditions consists of transition information on DataBackup's
                 Phase
               items:
-                description: DataBackupCondition describes conditions that explains
-                  transitions on phase
+                description: Condition explains the transitions on phase
                 properties:
                   lastProbeTime:
                     description: LastProbeTime describes last time this condition

--- a/charts/fluid/fluid/crds/data.fluid.io_dataloads.yaml
+++ b/charts/fluid/fluid/crds/data.fluid.io_dataloads.yaml
@@ -89,8 +89,7 @@ spec:
               description: Conditions consists of transition information on DataLoad's
                 Phase
               items:
-                description: DataLoadCondition describes conditions that explains
-                  transitions on phase
+                description: Condition explains the transitions on phase
                 properties:
                   lastProbeTime:
                     description: LastProbeTime describes last time this condition

--- a/charts/fluid/fluid/values.yaml
+++ b/charts/fluid/fluid/values.yaml
@@ -6,13 +6,13 @@ workdir: /tmp
 
 dataset:
   controller:
-    image: registry.cn-hangzhou.aliyuncs.com/fluid/dataset-controller:v0.6.0-6d1715f
+    image: registry.cn-hangzhou.aliyuncs.com/fluid/dataset-controller:v0.6.0-d04cd4d
 
 csi:
   registrar:
     image: registry.cn-hangzhou.aliyuncs.com/acs/csi-node-driver-registrar:v1.2.0
   plugins:
-    image: registry.cn-hangzhou.aliyuncs.com/fluid/fluid-csi:v0.6.0-6d1715f
+    image: registry.cn-hangzhou.aliyuncs.com/fluid/fluid-csi:v0.6.0-d04cd4d
   kubelet:
     rootDir: /var/lib/kubelet
 
@@ -21,9 +21,9 @@ runtime:
   alluxio:
     enabled: true
     init:
-      image: registry.cn-hangzhou.aliyuncs.com/fluid/init-users:v0.6.0-6d1715f
+      image: registry.cn-hangzhou.aliyuncs.com/fluid/init-users:v0.6.0-d04cd4d
     controller:
-      image: registry.cn-hangzhou.aliyuncs.com/fluid/alluxioruntime-controller:v0.6.0-6d1715f
+      image: registry.cn-hangzhou.aliyuncs.com/fluid/alluxioruntime-controller:v0.6.0-d04cd4d
     runtime:
       image: registry.cn-huhehaote.aliyuncs.com/alluxio/alluxio:release-2.5.0-1-SNAPSHOT-cb60e05
     fuse:

--- a/config/crd/bases/data.fluid.io_databackups.yaml
+++ b/config/crd/bases/data.fluid.io_databackups.yaml
@@ -106,8 +106,7 @@ spec:
               description: Conditions consists of transition information on DataBackup's
                 Phase
               items:
-                description: DataBackupCondition describes conditions that explains
-                  transitions on phase
+                description: Condition explains the transitions on phase
                 properties:
                   lastProbeTime:
                     description: LastProbeTime describes last time this condition

--- a/config/crd/bases/data.fluid.io_dataloads.yaml
+++ b/config/crd/bases/data.fluid.io_dataloads.yaml
@@ -89,8 +89,7 @@ spec:
               description: Conditions consists of transition information on DataLoad's
                 Phase
               items:
-                description: DataLoadCondition describes conditions that explains
-                  transitions on phase
+                description: Condition explains the transitions on phase
                 properties:
                   lastProbeTime:
                     description: LastProbeTime describes last time this condition

